libsel4/include/sel4/macros.h

```diff
@@ -33,14 +33,15 @@
 
 #ifdef CONFIG_LIB_SEL4_INLINE_INVOCATIONS
 #define LIBSEL4_INLINE static inline
-#else
-#define LIBSEL4_INLINE __attribute__((noinline)) __attribute__((unused)) __attribute__((weak))
-#endif
+#define LIBSEL4_INLINE_FUNC static inline
 
-#ifdef CONFIG_LIB_SEL4_PUBLIC_SYMBOLS
-#undef LIBSEL4_INLINE
+#elif CONFIG_LIB_SEL4_PUBLIC_SYMBOLS
```

Comment:
Uff, that means I have to come up with a prompt. And a "neither" option. *thinking*